<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>map</class>
 <widget class="QMainWindow" name="map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_adresspart2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>827</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInformations_2">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>643</y>
      <width>825</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Informations :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLatitude">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editLongitude">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>441</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>827</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
    <property name="source">
     <url>
      <string>qrc:/qml/mapview.qml</string>
     </url>
    </property>
   </widget>
   <widget class="QLabel" name="labelLongitude">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>440</y>
      <width>54</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chercher">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>827</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Location</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>821</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="labelPositionLatitude_2">
       <property name="text">
        <string>Latitude :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="positionLatitude_2">
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPositionLongitude_2">
       <property name="text">
        <string>Longitude :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="positionLongitude_2">
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="editLatitude">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>441</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonAller">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aller !</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
